internal/connector/cli_plugin.go

```diff
@@ -40,9 +38,33 @@ func (p *CliPlugin) Close() error {
 	} else {
 		p.logger.Infof("stdout pipe successfully closed")
 	}
+	if err := p.stderr.Close(); err != nil {
+		p.logger.Infof("failed to close stderr pipe")
+	} else {
+		p.logger.Infof("stderr pipe successfully closed")
+	}
```

Comment:
Is logging the fact that we successfully closed `stderr` worthwhile?  (I mean, that's the normal way of things, right?...so the only reason it might be interesting would be seeing it in sequence with other logged events...but...is even that interesting?  Otherwise, it's just noise and disk space.)

Likewise, is it actually useful to know that we were _unable_ to close `stderr`?  (Is that likely to happen?  How would we expect the user/developer to respond to seeing that message?  Does it support other information in the log?  Or, is it just more noise?  I.e., the only reason I see this happening would be if something else closed it first or if it was somehow never opened...which seem like they would be evident from other stuff in the log.)

So, I would be inclined to just call `Close()` and ignore the result....

Reply:
Yeah, I had that thought as well, but you should be able to quiet these log messages by changing the log level. Although, I don't know if that feature is currently working in the engine.

Reply:
> and you should be able to quiet these log messages by changing the log level.

Except that shutting off `info` level logging will lose most everything else, too, which I doubt most would actually consider a solution! Maybe they should be `debug` level, so they can be seen if someone really thinks the extra detail (of sequencing) is useful but normally (presumably) won't be shown?

Reply:
This is one that should probably be warning and debug, rather than info and info.

Reply:
we've really lost sight of the bug fix here, but yeah i'll change it

Reply:
@mfleader, my rule of thumb is that if the code is on a green background it's fair game for comment.  (That is, perhaps this request isn't directly applicable to the bug you intended to fix, but the concern arises from code which appears to have been introduced in the process of addressing that bug, and it would be best if the change didn't include anything disappointing.)

@jaredoconnell, why should it be a warning?  That is, what action would you expect a user to take in response to receiving the warning?  (I suspect that, if we find that we cannot close `stderr`, then I suspect that that is the least of our problems at that moment.)